kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveStores.java

```diff
@@ -161,12 +161,15 @@ public static WindowBytesStoreSupplier windowStoreSupplier(
       throw new IllegalArgumentException("Retention period cannot be less than window size");
     }
 
-    return new ResponsiveWindowedStoreSupplier(
+    final WindowBytesStoreSupplier ret = new ResponsiveWindowedStoreSupplier(
```

Comment:
just stashed these in a temp return variable so it's easy to re-enable these when ready